modules/user/user.resolver.js

```diff
@@ -7,14 +7,15 @@ const userQuery = {
 };
 const userMutation = {
   registerUser: (parent, args) => userService.registerUser(args.user),
-  loginUser: (parent, args) => userService.loginUser(args.user),
+  loginUser: (parent, args) => userService.loginUser(args.user, args.language),
```

Comment:
Revert this change, please, we are going to send back unlocalized error messages and localize them only on backend

Reply:
removed